nsqadmin/static/js/lib/handlebars_helpers.js

```diff
@@ -61,17 +61,24 @@ var genColorList = function(typ, key) {
     return 'blue';
 };
 
+// sanitizeGraphiteKey removes special characters from a graphite key
+// this matches behavior of bitly/statsdaemon
+// https://github.com/bitly/statsdaemon/blob/fc46d9cfe29b674a0c8abc723afaa9370430cdcd/statsdaemon.go#L64-L88
+var sanitizeGraphiteKey = function(s) {
+    return s.replaceAll(/[/ ]/g, '-').replaceAll(/[^a-zA-Z0-9-_.]/g, '')
```

Comment:
space should be replaced with underscore, I think

Reply:
(rephrased less confusingly: this replaces ` ` with `-`, but statsdaemon replaces ` ` with `_`)

Reply:
I think mreiferson's update changed this to replace both `/` and ` ` with `_`, but `/` still should be replaced with `-` :)